.circleci/config.yml

```
@@ -2,18 +2,18 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:9-stretch
+      - image: cimg/node:12.22
 
     branches:
-      only:
-        # Whitelist branches to build for.
-        - master
-        - build-test
+      # Whitelist branches to build for.
```

If we're switching `master` => `main` might as well go all the way.
```suggestion
      # Allowlist branches to build for.
```